channels/views/reports.py

```diff
@@ -0,0 +1,37 @@
+"""Views for REST APIs for reporting posts and comments"""
+
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from channels.serializers import ReportSerializer
+from channels.utils import translate_praw_exceptions
+
+
+class ReportContentView(APIView):
+    """
+    View to report a comment or post
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def get_serializer_context(self):
+        """Context for the request and view"""
+        return {
+            'request': self.request,
+            'view': self,
```

Comment:
it doesn't look like we're using this `view` property in the serializer anyway - do we need to add it for some behind-the-scenes thing?

Reply:
It was just from copy/paste of other views, I'll give removing it a try

Reply:
Removed it.